apps/web/src/components/borrow/modal/DepositModal.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import {
+  BoxLayout,
+  Button,
+  CenterLayout,
+  Icon,
+  Modal,
+  StackLayout,
+  Text,
+} from '@bloxifi/ui'
+import { BorrowAndLending, Tokens } from '@bloxifi/core'
+import { CheckAllowanceFunction } from '@bloxifi/types'
+import { useTranslation } from 'react-i18next'
+import { ethers } from 'ethers'
+
+import { TableInput } from '../table/TableInput'
+import { TransactionOverview } from '../table/TransactionOverview'
+
+import { Web3Container } from '@/containers/Web3Container'
+import { initailReserveData } from '@/containers/WalletContainer'
+
+interface Props {
+  /**
+   * Boolean value that defines if modal is open or closed
+   */
+  isOpen?: boolean
+  /**
+   * Function that defines closing of modal
+   */
+  onClose?: () => void
+  /**
+   * Selected asset reserve data
+   */
+  reserveData?: typeof initailReserveData
```

Comment:
this is a bit wrong, because if we change initialReserveData, this type will never break.
This needs to be `ReservesData`

and then let's see if `reserveData = initailReserveData` doesn't complain. And if we change initailReserveData this type will break, or if we change Type, we know that we need to change this initialReserveData.

Reply:
done